samples/matter/thermostat/src/thermostat.matter

```diff
@@ -1809,3 +1813,5 @@ endpoint 1 {
     handle command SetpointRaiseLower;
   }
 }
+
```

Comment:
I don't know why, but ZAP always adds two extra lines to this sample. You have to remove it manually from this file to fix the compliance check.